CHANGELOG.md

```diff
@@ -6,7 +6,9 @@
 
 ### Bug Fixes
 
-_None_
+* Improved format string regex to handle escaped "%"s that precede a variable
```

Comment:
Please use the same format as the other entries (with . and 2 spaces at the end of the description).